gradle/verification-metadata.xml

```diff
@@ -65,6 +65,7 @@
       </trusted-artifacts>
       <ignored-keys>
          <ignored-key id="74dafdfd6dae2441" reason="Key couldn't be downloaded from any key server"/>
+         <ignored-key id="39b48e1baddb933f" reason="Key couldn't be downloaded from any key server"/>
```

Comment:
🤔 Is this another dependency-verification glitch, that key was not downloaded?

Reply:
I tried to delete this entry and rerun `--write-verification-metadata` but got the same result. 🤷